src/wp-includes/widgets.php

```diff
@@ -1310,7 +1318,7 @@ function retrieve_widgets( $theme_changed = false ) {
 	$sidebars_widgets['wp_inactive_widgets'] = array_merge( $lost_widgets, (array) $sidebars_widgets['wp_inactive_widgets'] );
 
 	if ( 'customize' !== $theme_changed ) {
-		wp_set_sidebars_widgets( $sidebars_widgets );
+		wp_set_sidebars_widgets( $sidebars_widgets, false );
```

Comment:
A comment about why that "false" is snuck in there would be great!

Reply:
@draganescu 👍